tests/test_fetch_parq.py

```diff
@@ -544,7 +559,7 @@ def test_s3_parquet_to_dataframe(self):
         print(f"Key is: {key}")
 
         bucket = "foobucket"
-        key = "fookey/"
```

Comment:
so this sets the standard to no trailing slash right

Reply:
Yeah, fits in with the rest this way and I think this gets it all the way consistent from here I think